torch/utils/_cxx_pytree.py

```diff
@@ -61,6 +61,10 @@
 ]
 
 
+__TORCH_DICT_SESSION = optree.dict_insertion_ordered(True, namespace="torch")
+__TORCH_DICT_SESSION.__enter__()  # enable globally and permanently
```

Comment:
cc @zou3519

Reply:
If you're sure about this. The fun thing is if someone imports torch cxx_pytree and uses optree then they will override the optree behavior